sentry/migrations/20190806011140_initial-tables/seed/up.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO 
+channels 
+VALUES 
+(
+    '0x061d5e2a67d0a9a10f1c732bca12a676d83f79663a396f7d87b3e30b9b411088', 
+    '0x033ed90e0fec3f3ea1c9b005c724d704501e0196', 
+    '0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359', 
+    '1000',
```

Comment:
is it possible to use the native bigint here?

Reply:
We can't use BIGINT because it can only represent 64 bits integers while we need 256 bits integer representation